content/software-security/compliance/pci-dss-4/intro-pci-dss-4.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Introduction to the PCI Data Security Standard (DSS) 4.0"
+description: "How to prepare your organization to meet the requirements of PCI DSS 4.0"
+lead: "How to prepare your organization to meet the requirements of PCI DSS 4.0"
+type: "article"
+date: 2024-08-21T14:05:09+00:00
+lastmod: 2024-08-21T14:05:09+00:00
+contributors: []
+draft: false
+tags: ["compliance", "PCI DSS 4.0", "standards"]
+images: []
+menu:
+  docs:
+    parent: "pci-dss-4"
+weight: 001
+toc: true
+---
+
+PCI DSS 4.0, or PCI Data Security Standard, is a global standard in the payments industry that includes a set of foundational technical and operational requirements surrounding the protection of payment data. Its goal is to ensure the security of information involved when payment cards are used and while those payments are processed. PCI DSS 4.0 replaces the earlier PCI DSS 3.2.1, which was retired in March 2024.
```

Comment:
I think we should probably spell out Payment Card Industry here as well.

Reply:
Doh! Yes, we should.